Client/Frontend/Browser/URLBarView.swift

```diff
@@ -223,11 +238,23 @@ class URLBarView: UIView {
             make.size.equalTo(URLBarViewUX.TabsButtonHeight)
         }
 
+        tabsButton.snp_makeConstraints { make in
+            make.centerY.equalTo(self.locationContainer)
+            make.trailing.equalTo(self)
+            make.width.height.equalTo(UIConstants.ToolbarHeight)
```

Comment:
Its probably better to just make this stretch the height of the container (i.e. so it detection works regardless of if the toolbarheight changes). i.e. make.top.bottom.equalTo(self.locationContainer)